realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -326,6 +334,11 @@ public Builder deleteRealmIfMigrationNeeded() {
             return this;
         }
 
+        public Builder durability(SharedGroup.Durability durability) {
```

Comment:
Changed the public API to inMemory.

What would we do for the future ASYNC support? Add another method `async()`? And throw an exception if user call `async()` after calling `inMemory()`?